### ISSUE TITLE
Pokemon X/Y - Black spot on male character's face

### ISSUE THREAD

**Issue by Dragios** [citra-emu/citra#2133]
At first I thought this is related to this graphical bug #1515 but it seems like that one is fixed but not this one. Turning on/off shader JIT & using both hardware or software renderer does not change the result.

Citra (as of cc7f115)
![Black Spot Face](https://cloud.githubusercontent.com/assets/18496868/19618958/d7db784a-988b-11e6-8fae-c78f8e93f1cc.png)

3DS
![3DS reference](https://cloud.githubusercontent.com/assets/18496868/19619023/5ec6be08-988e-11e6-8877-e51377223154.png)

**Comment by wwylele** [comment]
Should mention that when the male character is on the screen, log spam with
```
HW.Memory <Error> core/memory.cpp:GetPointer:282: unknown GetPointer @ 0x00000000
```

**Comment by Subv** [comment]
The game renders the characters fine to the Color Buffer, but then enables and loads a NULL texture (Param 0x01002200: Address 0, Width 0, Height 0, Type 0, WrapT Repeat, WrapS Repeat, Filtering Linear) and draws the two black rectangles with it, one after the other.

**Comment by Subv** [comment]
Here's a command dump of the game drawing both spots:
https://gist.github.com/Subv/7bbfa381138e74fa5b5729dee5ac6764

The command list responsible for setting the texture addresses to 0 lives in physical address 0x2080A180. It doesn't appear to be a timing issue as the texture addresses are never replaced once the command list is stored in memory.

**Comment by Dragios** [comment]
Partially fixed by #3097 
Only software rasterizer is fixed.

**Comment by bamikhan** [comment]
Guys,has a solution to this been found out yet?
